queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
```

Comment:
**Changes Required**
- [ ] This query will not return records ending with `mon` but rather those containing `mon`. Let us try and fix this to only return records ending with `mon` as described in the requirement.

Reply:
Thank you for the suggestion, I have fixed it.